Two_Body_Guess/data/HF_mbed_scf_times.py

```diff
@@ -0,0 +1,10 @@
+{'Dimer-atom_g_scf_time': (9, 0.7873027324676514), 
```

Comment:
There are two geometries from the script that freeze up my computer: the second water hexamer, and the HF heptamer.

Reply:
The difference between the two water hexamers in the 'geometries' PR is explained in a README.